### PR TITLE
Home collections warning banner 855

### PR DESCRIPTION
**Pull request by ramavavi**
This PR dresses the issue : https://github.com/episphere/connect/issues/855

Code changes in this issue are:

1. Warning message banner should disappear after a few seconds - replace alertTemplate() in place of triggerErrorModal()
2.  Warning message "cup id/card id & supply kit/return kit are not the same" removed after submitting a kit.

### CODE REVIEW

File: src/pages/homeCollection/kitAssembly.js [hunk 1]
```diff
@@ -110,11 +110,14 @@ const enableEnterKeystroke = () => {
...
   const checkInputBox2 = document.getElementById(inputBox2);
+  if(errorMsg == "") {
+    document.getElementById(errorTag).innerHTML = `` 
+  }
   if (checkInputBox2) {
     checkInputBox2.addEventListener("input", (e) => {
       const checkInputBox2Value = e.target.value.trim();
       const checkInputBox1Value = document.getElementById(inputBox1).value.trim();
-      if (checkInputBox2Value !== checkInputBox1Value) {
+      if (checkInputBox2Value != checkInputBox1Value) {
```
Comment (Gbarra9) [inline src/pages/homeCollection/kitAssembly.js:120]:
```suggestion
      if (checkInputBox2Value !== checkInputBox1Value) {
```

The strict equality operator check may need to be added back.

**Review by Gbarra9 — APPROVED** [review]
Looks good

**Comment by ramavavi** [comment]
@abhinavjonnada82 @Gbarra9 thanks for input, will make changes, and update the code.

File: src/pages/homeCollection/kitAssembly.js [hunk 2]
```diff
@@ -110,6 +110,9 @@ const enableEnterKeystroke = () => {
 
 const performQCcheck = (inputBox2, inputBox1, errorTag, errorMsg) => {
   const checkInputBox2 = document.getElementById(inputBox2);
+  if (errorMsg == "") {
```
Comment (jhflorey) [inline src/pages/homeCollection/kitAssembly.js:113]: it should be `===`

**Review by jhflorey — APPROVED** [review]

**Review by jhflorey — APPROVED** [review]

File: src/pages/homeCollection/kitAssembly.js [hunk 3]
```diff
@@ -269,30 +272,32 @@ const storeAssembledKit = async (kitData) => {
...
     }
     else {
-      triggerErrorModal(`Failed to save the kit.`);
+      alertTemplate(`Failed to save the kit.`);
       return false
     }
   }
   else if (collectionUnique.data === 'Check Collection ID'){
-    triggerErrorModal('Check collection ID format.')
+    alertTemplate('Check collection ID format.')
```
Comment (we-ai) [inline src/pages/homeCollection/kitAssembly.js:280]: semicolon

File: src/pages/homeCollection/kitAssembly.js [hunk 4]
```diff
@@ -269,30 +272,32 @@ const storeAssembledKit = async (kitData) => {
...
     return false
   }
   else if (collectionUnique.data === 'duplicate collection id'){
-    triggerErrorModal('The collection card and cup ID are already in use.')
+    alertTemplate('The collection card and cup ID are already in use.')
     return false
   }
   else {
-    triggerErrorModal('Error')
+    alertTemplate('Error')
```
Comment (we-ai) [inline src/pages/homeCollection/kitAssembly.js:292]: semicolon

File: src/pages/homeCollection/kitAssembly.js [hunk 5]
```diff
@@ -269,30 +272,32 @@ const storeAssembledKit = async (kitData) => {
...
     }
   }
   else if (collectionUnique.data === 'Check Collection ID'){
-    triggerErrorModal('Check collection ID format.')
+    alertTemplate('Check collection ID format.')
     return false
   }
   else if (collectionUnique.data === 'duplicate supplykit id'){
-    triggerErrorModal('The supply kit and return kit are already in use.')
+    alertTemplate('The supply kit and return kit are already in use.')
```
Comment (we-ai) [inline src/pages/homeCollection/kitAssembly.js:284]: semicolon

File: src/pages/homeCollection/kitAssembly.js [hunk 6]
```diff
@@ -269,30 +272,32 @@ const storeAssembledKit = async (kitData) => {
...
     return false
   }
   else if (collectionUnique.data === 'duplicate collection id'){
-    triggerErrorModal('The collection card and cup ID are already in use.')
+    alertTemplate('The collection card and cup ID are already in use.')
```
Comment (we-ai) [inline src/pages/homeCollection/kitAssembly.js:288]: semicolon

**Review by we-ai — COMMENTED** [review]
Commented on minor things.

**Comment by ramavavi** [comment]
I have added the strict operators and the semicolons in the code.

**Review by we-ai — APPROVED** [review]

**Pull request merged**